Fix some formatting and also add uninstall instructions (#348)
ndless-nspire/Ndless: README.md

Prerequisites: OS 3.6

* Transfer `ndless_installer.tns` and `ndless_resources.tns` into a folder named "`ndless`" (top-level)
* Open the installer
* Press the menu button
* Have fun!

Uninstalling
------

> To uninstall Ndless, simply run the `ndless_resources.tns` file and select "`Yes`", after which, the device will reboot with Ndless uninstalled

Reviewer: ```suggestion
To uninstall Ndless, simply run the `ndless_resources.tns` file and select "Yes", after which the device will reboot.
```

  Geoffrey Anneheim aka geogeo - < geoffrey dot anneheim at free dot fr >
  Olivier Armand aka ExtendeD  - < olivier dot calc at gmail dot com >

Contributors and testers: Adriweb, bsl, critor, Excale, Goplat, hoffa, Jim Bauwens, Legimet, Levak, Lionel Debroux, lkj, tangrs... and many others.  

> You can see the full list of contributors [here](https://github.com/ndless-nspire/Ndless/graphs/contributors)

Reviewer: It's not entirely complete as it only has contributions after the move to git. Maybe just make it a `See also` kind of reference?

> You can also see the list of additionally GitHub contributors [here](https://github.com/ndless-nspire/Ndless/graphs/contributors)

Reviewer: ```suggestion
More contributions can be seen on [the GitHub repo](https://github.com/ndless-nspire/Ndless/graphs/contributors).
```
